pkg/storaged/btrfs/volume.jsx

```diff
@@ -92,33 +112,14 @@ const BtrfsVolumeCard = ({ card, block_devices, uuid, use }) => {
     // https://github.com/storaged-project/libblockdev/issues/966
     const is_mounted = btrfs_is_volume_mounted(client, block_devices);
 
-    function rename_dialog() {
-        dialog_open({
-            Title: _("Change label"),
-            Fields: [
-                TextInput("name", _("Name"),
-                          {
-                              validate: name => validate_fsys_label(name, "btrfs"),
-                              value: label
-                          })
-            ],
-            Action: {
-                Title: _("Save"),
-                action: function (vals) {
-                    return block_btrfs.SetLabel(vals.name, {});
-                }
-            }
-        });
-    }
-
     return (
         <StorageCard card={card}>
             <CardBody>
                 <DescriptionList className="pf-m-horizontal-on-sm">
                     <StorageDescription title={_("Label")}
                                                 value={label}
                                                 action={
-                                                    <StorageLink onClick={rename_dialog}
+                                                    <StorageLink onClick={() => rename_dialog(block_btrfs, label)}
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-20012-20240213-221034-6d18bdb8-fedora-39-devel//Coverage/lcov/github-pr.diff.gcov.html)